src/sdk/constants.ts

```diff
@@ -0,0 +1 @@
+export const KERNEL_PUBLIC_KEY = "";
```

Comment:
Doesn't seem right.

Reply:
This should have a value that we didn't create already, cf comment: https://github.com/ubiquity/ubiquibot-kernel/pull/64#issuecomment-2256249515

Reply:
So then we should rotate the key. I need to generate a new public key now.

Reply:
```
-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAo1AYSyrXTEjMj2USno6i
5dv9Da386RI/zGypmkWa1KrIspE/Yd8RPaaEVAwt7p6+YeGcOQVLruuk11fJxff1
xB+KGbk1+kIdQ7s70B7yVRZuIM/k5aGfPpeerm0wjt4dKcYTRrl/OjLOMRrZ3vCX
E96v6eHEOpZIJ9VnjzGA0xymc+kBuEmZKabuK16S9a7I+CkZC8unqXWi15Chlyw8
EmbquwSmXI8VJ4teUjsF/H1MhJK3WTfsdr8bDsooRTMVKVBL8jONpGPzJZtr39zY
dkRj2Je2kag9b3FMxskv1npNSrPVcSc5lGNYlnZnfxIAnCknOC118JjitlrpT6wd
8wIDAQAB
-----END PUBLIC KEY-----
```